stable/dex/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: dex
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: 0.1.0
+dependencies:
+- name: app
+  version: "0.4.0"
```

Comment:
this is the latest app chart?

Reply:
yepp, for now 😄 
https://github.com/odpf/charts/blob/main/stable/app/Chart.yaml#L18